site/docs/getinvolved.md

```diff
@@ -1,11 +1,38 @@
 ---
-title: Get Involved
+title: Get Involved with Quarks
 ---
 
-# Get Involved with Quarks
-
 Welcome to the Quarks community!  We would love to have you participate as a user of Quarks, a contributor or both.
 
+You can: 
+
+*   Use our project and give us feedback.
+*   Report bugs and submit patches.
+*   Contribute code, documenation, javadocs, and use-case ideas.
+
+## Mailing Lists
+
+You can ask for help and get involved with Quarks through our mailing lists:
+
+*   dev@quarks.incubator.apache.org is for people who want to contribute code to Quarks
+*   commits@quarks.incubator.apache.org is for commit messages and patches to Quarks
+
+To subscribe, send e-mails to dev-subscribe@quarks.incubator.apache.org and commits-subscribe@quarks.incubator.apache.org. 
+
+## Issue Tracking
+
+We use Jira here: [https://issues.apache.org/jira/browse/QUARKS](https://issues.apache.org/jira/browse/QUARKS)
+
+## GitHub Mirrors
+
+ Quarks source code: [https://github.com/apache/incubator-quarks](https://github.com/apache/incubator-quarks)
+ 
+ Website source: [https://github.com/apache/incubator-quarks-website](https://github.com/apache/incubator-quarks-website)
+ 
+## Developer Guidelines
+
+Check out our developer guidelines here: https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md
+
 ## Contribute Code
 
 If you plan to make any significant contribution, you will need to have a Contributor License Agreement [(CLA)](https://github.com/quarks-edge/quarks/blob/master/licences/Quarks-cla-individual.pdf) on file.  Get a copy of the CLA, fill it out and follow the instructions in the CLA to file it.  Committers will require a CLA before they commit contributions.
```

Comment:
I think everything from Contribute Code onward needs to be updated to reflect being at Apache. I don't know much of the information so it might be best to merge this and then have someone else edit. I think @kathyssaunders wrote the original document, so could maybe help with updating this?